lib/rubycritic/source_control_systems/git.rb

```diff
@@ -52,7 +52,13 @@ def self.switch_branch(branch)
       end
 
       def self.uncommitted_changes?
-        `git ls-files --other --exclude-standard --directory`.empty? && `git status --porcelain`.empty?
+        git_ls_files = `git ls-files --other --exclude-standard --directory`
+        git_status = `git status --porcelain`
+        if !git_ls_files.empty? || !git_status.empty?
+          puts "There are uncommitted changes, FYI..."
+        end
+        # git_ls_files.empty? && git_status.empty?
+        true
```

Comment:
not sure I understand why this was changed. 
We can't just return true all the time, and the puts message is not needed since the caller `switch_branch` already aborts with a similar message.

We cannot do a `git checkout` when there are uncommitted changes.

Reply:
Oops, I think I forgot to branch out before pushing some debugging changes. Although, while debugging I found out that `uncommitted_changes?` returns the opposite of what it means. It should be `!git_ls_files.empty? || !git_status.empty?`, or the method should be renamed something like `no_uncommited_changes?`